src/variants.php

```diff
@@ -823,6 +824,31 @@ function(sData) {
                 $_POST['type'] = NULL;
             }
 
+            // Add the source of the variant if a source can be found.
+            if (empty($_POST['source'])) {
+                // If nothing is given, the source is set to null, meaning that the source is unknown.
+                unset($aFieldsGenome[array_search('source', $aFieldsGenome)]);
+
+            } elseif ($_POST['source'] == 'VOT') {
+                // If the source is not a genome, but there is something given, we know it is a transcript.
+                // A transcript is described by adding an empty string as the source of the variant.
+                $_POST['source'] = '';
+
+            } else {
+                // If the source of the variant is not a transcript, it is a genome build.
+                //  We will then send the ID of this genome build to the database.
+                $sColumnSuffix = substr(strrchr($_POST['source'], '/'), 1);
+                if ($sColumnSuffix == 'DNA') {
+                    $sColumnSuffix = '';
+                }
```

Comment:
Volgens mij is deze code niet meer nodig. Hadden we dit niet gedaan vóór we foutmeldingen kregen dat `source` te lang was?